var/spack/repos/builtin/packages/pmdk/package.py

```diff
@@ -18,6 +18,7 @@ class Pmdk(Package):
     git      = "https://github.com/pmem/pmdk.git"
 
     version('develop', branch='master')
```

Comment:
If you want you can actually rename this version to `master` now that #1983 has been merged.

Reply:
@adamjstewart what is the recommended convention now?  I don't have a preference on the name myself, other than it is inconvenient to remember which packages are using `master` and which are using `develop` for this sort of thing.  If the consensus is to use `master` for the latest master branch then I'll push an update to this PR.

Reply:
There isn't really a consensus. My personal opinion is that we shouldn't be renaming branches that the developers themselves named. But I also never install anything from master/develop so I've never had to remember which package uses which name.